src/Hosting.Services.Web/Middlewares/ObsoleteCorrelationHeadersMiddleware.cs

```diff
@@ -120,6 +120,7 @@ private void ExtractCorrelationFromRequest(HttpRequest request)
 			"X-MS-Correlationid",
 			"MS-CorrelationId",			// Correlation Id header used by other Microsoft services
 			"corr",						// Correlation query parameter
+   			"correlationId",
```

Comment:
Why is there a need to add another parameter here?
Why can't the corr parameter just be used?

Reply:
rex is getting passed correlationId from partners and it would cause downtime to change it to corr on their side and our side while the current experiments are running. We need this logged to track and issue

Reply:
Approving on the premise that this is temporary and is removed in quick order.